File: Visual Studio Code/Recursion/Python/recursion_demo.py
'''
Programmer:     Corey Jenkins
Date:           January 27, 2022
Program Name:   recursion_demo.py
Purpose:        A demonstration on using a recursion in Python.
'''

def main():
    # We are passing the argument 5 to the message module.
    # We are telling it to display it five times.
    message(5)
    
def message(times):
    if(times > 0):
        print("This is a recursive modules.")
        message(times - 1)
main()

File: Logic & Design/Visual Studio Code/Recursion/Java/RecursionDemo.java
public class RecursionDemo
{
	public static void main(String[] args)
	{
		// By passing the argument 5 to the message method
		// we are telling it to display the message 5 tmes.
		message(5);
	}

	public static void message(int n)
	{
		if (n > 0)
		{
			System.out.println("This is a recursive method.");
			message(n - 1);
		}
	}
}